app/javascript/components/widgets/widgets/land-cover/tree-cover/selectors.js

```diff
@@ -62,6 +62,7 @@ export const getSentence = createSelector(
     const params = {
       year: settings.extentYear,
       location: currentLabel || 'global',
+      location_alt: `${currentLabel}'s`,
```

Comment:
Ideally when it comes to translations, this `'s` should be in the sentence so that it can be changed.